lib/recurly/apple-pay.js

```diff
@@ -8,6 +8,16 @@ const debug = require('debug')('recurly:apple-pay');
 
 const SESSION_TIMEOUT = 300000; // 5m
 const APPLE_PAY_API_VERSION = 2;
+const BILLING_DATA_MAP = {
```

Comment:
let's rename this `APPLE_PAY_ADDRESS_MAP` or something similar to be more specific.

Reply:
I thought about something like that, but having Apple Pay in the name in the Apple Pay class seemed redundant.